src/Compilers/CSharp/Test/Emit2/Diagnostics/DiagnosticAnalyzerTests.cs

```diff
@@ -516,6 +516,7 @@ public override void Initialize(AnalysisContext context)
             }
         }
 
+#pragma warning disable CS0618 // Type or member is obsolete - IsDiagnosticAnalyzerSuppressed is obsolete
```

Comment:
We are retaining the existing functionality and unit tests for the deprecated API as per the feedback: https://github.com/dotnet/roslyn/issues/67592#issuecomment-1507646644

Reply:
I wasn't sure if tests marked with Obsolete attribute would be executed or skipped by the test runner.